modules/ai/assets/js/editor/pages/form-layout/hooks/use-screenshots.js

```diff
@@ -44,16 +42,33 @@ const useScreenshots = ( { onGeneration } ) => {
 						const updatedData = [ ...prev ];
 						const placeholderIndex = updatedData.lastIndexOf( PLACEHOLDER_VALUE );
 
-						updatedData.splice( placeholderIndex, 1 );
+						updatedData[ placeholderIndex ] = { isPlaceholder: true };
```

Comment:
you have `PLACEHOLDER_VALUE` and `{ isPlaceholder: true }`
what's the difference? maybe change the name to one f them?

Reply:
Done.